opengrok-indexer/src/main/java/org/opengrok/indexer/web/PageConfig.java

```diff
@@ -923,10 +923,6 @@ private static void splitByComma(String value, List<String> result) {
         TreeSet<String> projectNames = new TreeSet<>();
         List<Project> projects = getEnv().getProjectList();
 
-        if (projects == null) {
-            return projectNames;
-        }
-
```

Comment:
why is this no longer needed?

Reply:
`projects` is always defined because `RuntimeEnvironment` does a `new ArrayList`.

Reply:
ok, nice